src/components/Home/Home.js

```diff
@@ -1,12 +1,9 @@
 import React from 'react'
```

Comment:
I have become quite attached to the Home app's initial animation, so not seeing it felt a bit abrupt 😅.

Reply:
I know! :laughing: It was feeling a bit too much after the loading screen transition, so I decided to remove it.